devtools/release/update_default_valid_target.sh

```diff
@@ -43,17 +43,29 @@ function get_60days_ago_block(){
 }
 
 function print_60_days_ago_block(){
-  local host=$1
+  local network=$1
+  local host=$2
+  local explorer_url=$3
+  
   ASSUME_TARGET_HEIGHT=$(get_60days_ago_block ${host})
+  ASSUME_TARGET_HEIGHT_DECIMAL=$(printf "%d" ${ASSUME_TARGET_HEIGHT})
   ASSUME_TARGET_HASH=$(get_block_hash ${host} ${ASSUME_TARGET_HEIGHT})
   ASSUME_TARGET_TIMESTAMP=$(get_block_timestamp ${host} ${ASSUME_TARGET_HEIGHT})
   ASSUME_TARGET_DATE=$(date -d @$((${ASSUME_TARGET_TIMESTAMP} / 1000)))
-  printf "the 60 days ago block is: %d %s in %s\n" ${ASSUME_TARGET_HEIGHT} ${ASSUME_TARGET_HASH} "${ASSUME_TARGET_DATE}"
+  EXPLORER_URL=${explorer_url}/block/${ASSUME_TARGET_HASH}
+  printf "the 60 days ago block is: %d %s in %s\n" ${ASSUME_TARGET_HEIGHT_DECIMAL} ${ASSUME_TARGET_HASH} "${ASSUME_TARGET_DATE}"
+  printf "you can view this block in ${EXPLORER_URL}\n\n"
+
+  TEXT="    // Default assume valid target for ${network}, expect to be a block 60 days ago.\n    // Need to update when CKB's new release\n    // in ${network}: the 60 days ago block is:\n    // height: ${ASSUME_TARGET_HEIGHT_DECIMAL}\n    // hash: ${ASSUME_TARGET_HASH}\n    // date: ${ASSUME_TARGET_DATE}\n    // you can view this block in ${EXPLORER_URL}\n    pub const DEFAULT_ASSUME_VALID_TARGET: &str =\n        \"${ASSUME_TARGET_HASH}\";"
+
+sed -i "/pub mod ${network} {/,/}/c\pub mod ${network} {\n${TEXT}\n}" util/constant/src/default_assume_valid_target.rs
```

Comment:
`sed -i` does not work for freebsd sed used by macOS.

Use `sed -i.bak ` then remove the file `util/constant/src/default_assume_valid_target.rs.bak`

```suggestion
  sed -i.bak "/pub mod ${network} {/,/}/c\pub mod ${network} {\n${TEXT}\n}" util/constant/src/default_assume_valid_target.rs
  rm -f util/constant/src/default_assume_valid_target.rs.bak
```

Reply:
OK